internal/view/cmd/args.go

```diff
@@ -57,7 +57,7 @@ func newArgs(p *Interpreter, aa []string) args {
 		default:
 			switch {
 			case p.IsContextCmd():
-				arguments[contextKey] = a
+				arguments[contextKey] = strings.Join(aa, " ")
```

Comment:
Tho this will work we also need to handle this cmd if we allow spaces aka `po @Denv Env ns1 app=duh`